lib/roby/app.rb

```diff
@@ -850,7 +852,13 @@ def load_base_config
             end
         end
 
+        def base_setup_done?
+            @base_setup_done
+        end
+
         def base_setup
+            return if base_setup_done?
```

Comment:
`raise` or warn and return. Don't ignore silently.

Probably best to warn and return to avoid breaking things unexpectedly.